### DISCUSSION TITLE
Notify.create is not a function

### DISCUSSION THREAD

**Discussion by pytely** [quasarframework/quasar#12274] [General - Components / Directives / etc] [answered]
Uncaught (in promise) TypeError: quasar_src_plugins_Notify_js__WEBPACK_IMPORTED_MODULE_0__.default.create is not a function
    at warning (webpack-internal:///../../../src/helpers/alarms.js:17:72)
    at eval (webpack-internal:///./src/store/rsc/actions/init.js:92:64)
the code for alrams.js is:

```
import { Notify } from 'quasar'
export function warning (msg) {
  Notify.create('msg')
}

```
quasar.conf.js plugins section:

```
      plugins: [
        'Notify',
        'Loading',
        'Dialog'
      ]

```
quasar.conf.js webpack sections:

```
      extendWebpack (cfg) {
        cfg.module.rules.push({
          resourceQuery: /blockType=i18n/,
          type: 'javascript/auto',
          loader: '@intlify/vue-i18n-loader',
          exclude: /(node_modules|quasar)/
        })
      },
      chainWebpack (chain) {
        const nodePolyfillWebpackPlugin = require('node-polyfill-webpack-plugin')
        chain.plugin('node-polyfill').use(nodePolyfillWebpackPlugin)
        chain.plugin('eslint-webpack-plugin')
          .use(ESLintPlugin, [{ extensions: ['js', 'vue'] }])
        chain.resolve.alias
          .set('helpers', path.resolve(__dirname, './src/helpers'))
        chain.resolve.alias
          .set('router', path.resolve(__dirname, './src/router'))
        chain.resolve.alias
          .set('mixins', path.resolve(__dirname, './src/mixins'))
        chain.resolve.alias
          .set('services', path.resolve(__dirname, './src/services'))
        chain.resolve.alias
          .set('palette', path.resolve(__dirname, './src/palette'))
      }
    },

```
Where I'm wrong?

**Comment by rstoenescu** [comment, marked as answer]
Have you installed the Notify plugin? quasar.conf > framework > plugins: [ 'Notify' ]

**Reply by pytely** [reply]
Yes, I did. 
My plugins configuration is:
 ```
     plugins: [
        'Notify',
        'Loading',
        'Dialog'
      ]
```
I tested Dialog as well, it does not work also if I use Dialog.create.
However, $q.dialog in setup() and this.$q.dialog in methods: {} does work

**Reply by rstoenescu** [reply]
Create a codesandbox.

**Reply by pytely** [reply]
I'll do.
But it looks like I found the cause of the problem.
I created a helper module. alarms.js.
```
import { Notify } from 'quasar'
export function warning (msg) {
  Notify.create('msg')
}
```
I used this function in vuex actions including it via: 
`include { warning } from 'helpers/alarms.js'`

It looks like this helpers must be loaded in boot. 
When I put Notify directly to action, everything works fine.
